src/commands/moderation/prefixCommands/functions/modifyVersion.ts

```diff
@@ -0,0 +1,166 @@
+import { ChatInputCommandInteraction, Colors, TextChannel, User } from 'discord.js';
+import { constantsConfig, getConn, PrefixCommandVersion, Logger, makeEmbed, refreshSinglePrefixCommandVersionCache, PrefixCommand } from '../../../../lib';
+
+const noConnEmbed = makeEmbed({
+    title: 'Prefix Commands - Modify Version - No Connection',
+    description: 'Could not connect to the database. Unable to modify the prefix command version.',
+    color: Colors.Red,
+});
+
+const failedEmbed = (versionId: string) => makeEmbed({
+    title: 'Prefix Commands - Modify Version - Failed',
+    description: `Failed to modify the prefix command version with id ${versionId}.`,
+    color: Colors.Red,
+});
+
+const wrongFormatEmbed = (invalidString: string) => makeEmbed({
+    title: 'Prefix Commands - Modify Version - Wrong format',
+    description: `The name and alias of a version can only contain alphanumerical characters, underscores and dashes. "${invalidString}" is invalid.`,
+    color: Colors.Red,
+});
+
+const doesNotExistsEmbed = (version: string) => makeEmbed({
+    title: 'Prefix Commands - Modify Version - Does not exist',
+    description: `The prefix command version ${version} does not exists. Can not modify it.`,
```

Comment:
```suggestion
    description: `The prefix command version ${version} does not exists. Cannot modify it.`,
```
"Cannot" is generally more common. I suggest we use that but obviously that's personal preference and I'm happy to leave it as is!